README.md

```diff
@@ -17,6 +17,7 @@ This theme is an upgraded version of the [Dream Theme](https://github.com/g1eny0
 * Custom CSS and JS can be used without modifying the theme
 * More social icons
 * Shorte.st website script, and a lot of other minor improvements.
+* RSS in sidebar
```

Comment:
I don't think it's a feature, so please remove this line from here. I'll document it in the Readme before merging your PR.

Reply:
ok, i will remove it.